docs/data-sources/mws_network_connectivity_config.md

```diff
@@ -0,0 +1,47 @@
+---
+subcategory: "Deployment"
+---
+# databricks_mws_network_connectivity_config Data Source
+
+-> **Note** If you have a fully automated setup with workspaces created by [databricks_mws_workspaces](../resources/mws_workspaces.md) or [azurerm_databricks_workspace](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/databricks_workspace), please make sure to add [depends_on attribute](../guides/troubleshooting.md#data-resources-and-authentication-is-not-configured-errors) in order to prevent _default auth: cannot configure default credentials_ errors.
+
+Lists all [databricks_mws_network_connectivity_config](../resources/mws_network_connectivity_config.md) in Databricks Account.
+
+-> **Note** [`account_id`](../index.md#account_id) provider configuration property is required for this resource to work.
+
+## Example Usage
+
+Fetching information about a network connectivity configuration in Databricks Account
+
+```hcl
+provider "databricks" {
+  // other configuration
+  account_id = "<databricks account id>"
+}
+
+data "databricks_mws_network_connectivity_config" "this" {
+  network_connectivity_config_id = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
```

Comment:
Added `databricks_mws_network_connectivity_configs` to support region filtered look-up and `databricks_mws_network_connectivity_config` will now only accept `name` attribute rather than `network_connectivity_config_id`